geometry/optimization/iris.cc

```
@@ -618,14 +620,19 @@ HPolyhedron IrisInConfigurationSpace(const MultibodyPlant<double>& plant,
                 .IntersectsWith(*obstacles[scaling[i].second])) {
           const VectorXd point = closest_points.col(scaling[i].second);
           AddTangentToPolytope(E, point, 0.0, &A, &b, &num_constraints);
+          if (options.require_sample_point_is_contained) {
```

I guess the change here is only for performance (fail fast)